### PR TITLE
Add Footer

### LINKED ISSUE #7: Add GitHub link

### PR DESCRIPTION
**Pull request by bash**
Resolves #7

### CODE REVIEW

File: client/less/components/page-footer.less [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.page-footer {
+    background-color: var(--background-level-1-color);
+    box-shadow: var(--background-level-1-box-shadow);
+    padding: 38px;
+    font-size: 14px;
+
+    > a {
+        text-decoration: underline;
+    }
+}
```
Comment (jnferner) [inline client/less/components/page-footer.less]: Missing newline

File: client/less/components/page-layout.less [hunk 2]
```diff
@@ -0,0 +1,9 @@
+.page-layout {
+    display: flex;
+    flex-direction: column;
+    min-height: 100vh;
+
+    > .content {
+        flex-grow: 1;
+    }
+}
```
Comment (jnferner) [inline client/less/components/page-layout.less]: Missing newline

File: client/index.html [hunk 3]
```diff
@@ -7,7 +7,19 @@
...
+    <main class="content" id="js-main-content"></main>
+    <div class="footer">
+        <div class="page-footer">
+            This site is developed publicly on <a href="https://github.com/isitvegan/isitvegan-web">GitHub</a>
+            where you can <a href="https://github.com/isitvegan/isitvegan-web/issues/new">report issues</a>.
+            <br />
+            <br />
+            © 2019 <a href="https://www.jnferner.com">Jan Nils Ferner</a> and
+                   <a href="https://www.rubys.ninja">Ruben Schmidmeister</a>.
+                For content see individual sources.
```
Comment (jnferner) [inline client/index.html]: I think we can cut this.

File: client/less/main.less [hunk 4]
```diff
@@ -28,3 +29,5 @@
 @import './components/search-scope-bar.less';
 @import './components/search-result-item-status.less';
 @import './components/links-list.less';
+@import './components/page-layout.less';
+@import './components/page-footer.less';
```
Comment (jnferner) [inline client/less/main.less]: Missing newline

File: client/index.html (hunk 3 above)
Comment (bash) [reply client/index.html]: 8998f32

**Pull request merged**